### ISSUE TITLE
Metagenomic, metabolomic, and lipidomic shifts associated with fecal microbiota transplantation for recurrent Clostridioides difficile infection

### ISSUE THREAD

**Issue by SvetlanaUP** [waldronlab/BugSigDBcuration#511]
Metagenomic, metabolomic, and lipidomic shifts associated with fecal microbiota transplantation for recurrent Clostridioides difficile infection – Arthur S. McMillan – mSphere
https://journals.asm.org/doi/10.1128/msphere.00706-24

**Comment by Zzee2** [comment]
Hi @SvetlanaUP i would love to take this up. could you assign to me. thanks

**Comment by PATIENCE-ONAH** [comment]
Hi @Zzee2 , how is your curation going?. I'm available to collaborate with you. Do let me know if you need any help.

**Comment by SvetlanaUP** [comment]
@PATIENCE-ONAH you got your assignment, please check https://github.com/waldronlab/BugSigDBcuration/issues/354

**Comment by PATIENCE-ONAH** [comment]
Yes, I have already commented before I saw your reply. I have taken up the work.
Thank you @SvetlanaUP

**Comment by AishatBolarinwa** [comment]
Hello @Zzee2  How is this curation going? Would you like me to assist in any way?

You can find me on the slack channel @Bolarinwa Aishat

**Comment by KateRasheed** [comment]
Good morning @Zzee2 . Trust you are doing well. I entered the two PMIDs of your article and noticed you are yet to create a study. Are you experiencing any blocker? Please feel free to reach out to us for assistance as "support systems". We are here to support you in any way.

**Comment by SvetlanaUP** [comment]
@KateRasheed what do you mean by "two PMIDs"? Every paper has only one PMID.

**Comment by KateRasheed** [comment]
![Screenshot_20241024-084519](https://github.com/user-attachments/assets/5a94ad11-a633-4471-99cd-538f55e28071)
@SvetlanaUP I took a screenshot.

**Comment by SvetlanaUP** [comment]
@KateRasheed first one is the study published as a preprint (bioRxiv) and the second one is one that we curate: paper published in the peer reviewed journal (mSphere)

**Comment by SvetlanaUP** [comment]
@KateRasheed @AishatBolarinwa please do together this curation, thanks!

**Comment by AishatBolarinwa** [comment]
Noted ma'am @SvetlanaUP 
I'll reach out to them right away.

Thank you

**Comment by AishatBolarinwa** [comment]
This is insightful.
Thank you.

**Comment by KateRasheed** [comment]
Thank you @SvetlanaUP. We'll get to work right away.

**Comment by AishatBolarinwa** [comment]
> @KateRasheed @AishatBolarinwa please do together this curation with @Zzee2 , thanks!

Thank you 

@Zzee2 Kindly drop your slack username so we can reach out to you. Thank you.

**Comment by KateRasheed** [comment]
Good day @SvetlanaUP . This study is ready for review 

https://bugsigdb.org/Study_1186

@AishatBolarinwa

**Comment by AleruDivine** [comment]
Well done on this curation @Zzee2 @AishatBolarinwa @KateRasheed 
Just a couple of things for you to look at.

1. Lactobacillaceae in Signature 2 of experiment 4 is [Lactobacillaceae](https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Tree&id=33958&lvl=3&keep=1&srchmode=1&unlock) Taxonomy ID: 33958
2. The signatures in Figure 1F was added to experiment 4, this is a random forest analysis of species identified through metagenomics of stool samples. Please confirm if this should be curated. If yes, should be curated as a different experiment

Thank you so much!

**Comment by KateRasheed** [comment]
Thanks for the review @AleruDivine . Figure 1F should not be curated and I've removed it from the signatures. 

All corrections have been made on the study. Thank you.

**Comment by AleruDivine** [comment]
Thank you @KateRasheed!

This review is complete @SvetlanaUP

**Comment by AleruDivine** [comment]
Thank you @KateRasheed!

This review is complete @SvetlanaUP.

**Comment by SvetlanaUP** [comment]
https://bugsigdb.org/Study_1186 reviewed.
